Enhancement: standard mark down docs (#1671)
vegastrike/Vega-Strike-Engine-Source: SUPPORT.md

# Support

The Vega Strike Development team has the following support policies:

- Please file an issue in our GitHub Issues as this is the fastest way to get attention from the development team.

> - We do try to monitor various please:

Reviewer: ```suggestion
- We do try to monitor the following:
```

Not sure exactly how this was supposed to read. This is my best guess.

   - Gitter.Im/Matrix Vega Strike Community
   - The Vega Strike Forums
   - Vega Strike IRC Channel
   - Various Vega Strike Emails and Social Media accounts

> However we cannot be every where. We do ask that you participate in GitHub where the team is centralized.

Reviewer: ```suggestion
However we cannot be everywhere. We do ask that you participate in GitHub where the team is centralized.
```

We do, however, want to hear from you. Please coordinate with us via the Vega Strike Forums to get your patches accepted through an alternate means.

## What constitutes a supported platform?

We cannot support every single platform; however, GitHub has made it easy for us to support quite a few via our CI/CD systems.

We do support the three major desktop platforms: Windows, Mac, and Linux
As a general guide please visit https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Platform-Support for our support policies.

Some of our technological decisions are based on the compiler and library support on our supported platforms; however,
we do reserve the right to require newer build tools than what may be on a given platform if we are able to make it work in our CI system.

### Linux

The development team is primarily on Linux.
We try to support currently supported platforms for the major distributions:
- Red Hat/Fedora/Rocky Linux
- OpenSuSe
- Debian/Ubuntu/LinuxMint

Please visit https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Platform-Support#linux-distribution-support-policy for our Linux Support Policy.

> ### MacOS

Reviewer: ```suggestion
### macOS
```

> We have supported MacOS releases since version 0.9.0 when we were able to build a DMG images again.

Reviewer: ```suggestion
We have supported macOS releases since version 0.9.0 when we were able to build DMG images again.
```

> As a general policy we support the current versions of MacOS as long as we have access to a builder in GitHub.

Reviewer: ```suggestion
As a general policy we support the current versions of macOS as long as we have access to a builder in GitHub.
```

Please visit https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Platform-Support#macos for our macOS Support Policy.

### Windows

We have supported Windows releases since version 0.9.0 when we were able to generate NSIS Installers again.

As a general policy we support the current versions of Windows as long as we have access to a builder in GitHub.
However, if something were to fail then it may take us a while to fix it as the team generally does not have access to Windows development systems.

Please visit https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Platform-Support#windows for our Microsoft Windows Support Policy.

## Supported Releases

We have generally only maintained a single release at a time, and our primary focus in on our next deliverable.
If there are bugs in an existing current release then we will address it and make a new point release.

| Release Version | Status | Release Date | EOL Date | Comments |
| :-------------- | :----: | :----------: | :------: | :------- |
| 0.11.x | In Development | TBD | N/A | Current version in development |

> | 0.10.x | In Development | TBD | N/A | This will be our first LTS due to some code dependency changes that will effect some older systems |

Reviewer: ```suggestion
| 0.10.x | In Development | TBD | N/A | This will be our first LTS (Long Term Support) release due to some code dependency changes that will affect some older systems |
```
